go-mod-hack.sh

```diff
@@ -1,21 +1,24 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This script was a copy&paste from a github issue. Did you check if it's still working as expected?

Reply:
yep, i made sure it is working still :)